sstmac/software/process/thread.cc

```diff
@@ -358,7 +355,9 @@ Thread::computeDetailed(uint64_t flops, uint64_t nintops, uint64_t bytes, int nt
 }
 
 void
-Thread::collectStats(Timestamp start, TimeDelta elapsed)
+Thread::collectStats(
+    SSTMAC_MAYBE_UNUSED Timestamp start, 
```

Comment:
These macros were giving me warnings on GCC 7

Reply:
Same as before.

Reply:
Build is clean. SSTMAC_MAYBE_UNUSED used to generate errors on devel.